draft-ietf-quic-tls.md

```diff
@@ -541,7 +541,7 @@ older than 1.3 is negotiated.
 
 QUIC requires that the initial handshake packet from a client fit within the
 payload of a single packet.  The size limits on QUIC packets mean that a record
-containing a ClientHello needs to fit within 1197 octets.
+containing a ClientHello needs to fit within 1165 octets.
```

Comment:
To be clear, this 1165 number may change if we increase or decrease the amount of framing overhead QUIC adds.  Are we ok about that?